python/mlx/nn/layers/normalization.py

```diff
@@ -6,6 +6,67 @@
 from mlx.nn.layers.base import Module
 
 
+class InstanceNorm(Module):
+    r"""Applies instance normalization [1] on the inputs.
+
+    Computes
+
+    .. math::
+
+        y = \frac{x - \mathrm{E}[x]}{ \sqrt{\mathrm{Var}[x] + \epsilon}} * \gamma + \beta,
+
+    where :math:`\gamma` and :math:`\beta` are learned per feature dimension
+    parameters initialized at 1 and 0 respectively. Both are of size dims,
+    if :attr:`affine` is ``True``.
+
+    [1]: https://arxiv.org/abs/1607.08022
+
+    Args:
+        dims (int): The number of features of the input.
+        eps (float): A value added to the denominator for numerical stability. Default: ``1e-5``.
+        affine (bool): A boolean value that indicates whether this module has learnable affine parameters,
+            initialized the same way as done for batch normalization.
+            Default: ``False``.
+    Shape:
+        - Input: :math:`(N, L, C)` or :math:`(N, H, W, C)` or :math:`(N, D, H, W, C)`
+        - Output: :math:`(N, L, C)` or :math:`(N, H, W, C)` or :math:`(N, D, H, W, C)` (same shape as input)
```

Comment:
I would simplify this to something like:

```
Shape:
      - Input: :math:`(..., C)` where :math:`C` is equal to ``dims``.
      - Output: Same shape as the input.
```

Reply:
Great suggestion.